### ISSUE TITLE
Custom Codes Hash Automation Replacement being duplicated after JCB update to 2.11.0

### ISSUE THREAD

**Issue by duetnet** [vdm-io/Joomla-Component-Builder#554]
### Steps to reproduce the issue*

Trying to replace code into admin view MODEL save($data) function source eg...

Original code reads
```
			$data['published'] = 0;
```

Replace with following code
```
/***[REPLACE<>$$$$]***/
			$data['published'] = 1;
/***[/REPLACE<>$$$$]***/
```

**### Expected result**
```
/***[REPLACED$$$$]***//*191*/
			$data['published'] = 1;
/***[/REPLACED$$$$]***/
```

### Actual result

Although PHP code reflects the following correctly, i.e
```
/***[REPLACED$$$$]***//*191*/
			$data['published'] = 1;
/***[/REPLACED$$$$]***/
```
The "custom codes" listing in the gui NOW shows 2 identical records (#190 and #191) for this replacement where all details (from line, to line, add type, end and start hash and path) are identical however the CUSTOM CODE section for 
- record 190 reflects the OLD code i.e. $data['published'] = 0;
- record 191 reflects the NEW code i.e. $data['published'] = 1;

### System information (as much as possible)
- OS Name & Version: Win10Pro
- MySql Version: 5.5.5-10.4.11-MariaDB 
- Apache Version: 2.4.41 win 64b
- PHP Version: 7.4.4 
- Joomla Version: 3.9.18
- JCB Version: 2.11.0
- Browser: Firefox 75.0 64b

### Additional comments

This "DUPLICATE RECORD" occurs with all my hash REPLACE statements, but not with INSERT statement.

If I recompile code again, no changes occur in the compiled code, it shows the replacedment correctly....

where every record is duplicated as per above...
If I delete the DUPLICATE OLD RECORD, eg. record 190 above, and recompile, the code still reflects the new changed coding..

So, what is the reason for this DUPLICATION as it creates many duplicates for every line fo code which has been replaced with the JCB REPLACE option?

Regards

Franz

**Comment by Llewellynvdm** [comment]
This is weird, I will try to take a closer look.

I have not had the same effect... but then I have not tested it recently.

**Comment by pctechnikch** [comment]
Yes, unfortunately it is so, all custom codes that are changed double. Every time you compile, a new one is written. Unfortunately, the custom code insertion method no longer works. since the last update. This is a pity .

**Comment by Llewellynvdm** [comment]
Will it be possible to make a video recording of this? because I do not see this.

**Comment by Llewellynvdm** [comment]
Then also please upgrade to v2.11.2

**Comment by duetnet** [comment]
I have just run upgrade from 2.10.13 to 2.11.2 and DUPLICATION is not happening anymore.
I have tried custom code with INSERT and REPLACE and all working correctly.
It seems that you have corrected the issue I encountered in 2.11.1
Issue can be closed, thank you.
